modules/ui/stickyNotes.js

```diff
@@ -0,0 +1,229 @@
+let version = '5.6.1';
```

Comment:
Version shouldn't start at 5.6.1?

Reply:
I was counting versions also when it was in development, so the previous version was correct.